README.md

```diff
@@ -47,6 +43,14 @@ In LoopBack applications, add the following entry to your
 }
 ```
 
+## Content Type
+Depending on the request header's `Accepts`, response will be returned in
```

Comment:
Please add an empty line after the `##` header (keep consistency with the rest of this file).